spec/codecs/netflow_spec.rb

```diff
@@ -2038,7 +2050,7 @@
             "netscalerHttpReqUserAgent": "Mozilla/5.0 (Commodore 64;  kobo.com) Gecko/20100101 Firefox/75.0",
             "destinationTransportPort": 443,
             "netscalerHttpReqCookie": "beer=123456789abcdefghijklmnopqrstuvw; AnotherCookie=1234567890abcdefghijklmnopqr; Shameless.Plug=Thankyou.Rakuten.Kobo.Inc.For.Allowing.me.time.to.work.on.this.and.contribute.back.to.the.community; Padding=aaaaaaaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbccccccccccccccddddddddddddddddddddddeeeeeeeeeeeeeeeeeeeeeffffffffffffffffffffffgggggggggggggggggggggggghhhhhhhhhhhhhhhhhiiiiiiiiiiiiiiiiiiiiiijjjjjjjjjjjjjjjjjjjjjjjjkkkkkkkkkkkkkkkkkklllllllllllllllmmmmmmmmmm; more=less; GJquote=There.is.no.spoon; GarrySays=Nice!!; LastPadding=aaaaaaaaaaaaabbbbbbbbbbbbbbbbbbbbcccccccccccccccccccdddddddddddeeeeeeee",
-            "flowEndMicroseconds": "2016-11-11T12:09:19.000Z",
+            "flowEndMicroseconds": "2016-11-11T12:09:19.000#{nanos}Z",
```

Comment:
Do you think it makes sense to have tests for `flowEndMicroseconds` and `flowEndNanoseconds`? I think we should at least have a test for µs precision as well as ms and ns

Reply:
I think that to insert that test we have to update the fixture data, in particular binary data files like:
- `ipfix_test_netscaler_tpl.dat`
- `ipfix_test_netscaler_data.dat`
which I don't know where they came from. So it's not clear to me if (and how) we can update those binary files.